core/controller/src/main/scala/whisk/core/controller/RestAPIs.scala

```diff
@@ -80,7 +80,7 @@ protected[controller] class SwaggerDocs(apipath: Uri.Path, doc: String)(implicit
 
 protected[controller] object RestApiCommons {
   def requiredProperties =
-    Map(WhiskConfig.servicePort -> 8080.toString) ++
+    Map(WhiskConfig.servicePort -> 8080.toString, WhiskConfig.controllerProtocol -> "https") ++
```

Comment:
Do we really want to set https to be the default here? It's harder to develop on, isn't it?